rules/use-a-signature-with-a-link-when-commenting-on-a-blog/rule.md

```diff
@@ -1,30 +1,53 @@
 ---
 type: rule
-title: Do you use a signature with a link when commenting on a blog?
-uri: use-a-signature-with-a-link-when-commenting-on-a-blog
+title: Do you use signature links and monitor GitHub stars for online engagement?
+uri: use-sig-links-github-stars
```

Comment:
Please also update this rule URL in its category Rules to Better Blogging 🙏

Reply:
Done - good catch 🙏